code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -503,6 +503,7 @@
 	id = "bananium golem"
 	fixed_mut_color = "ff0"
 	say_mod = "honks"
+	inherent_traits = list(TRAIT_RESISTHEAT,TRAIT_NOBREATH,TRAIT_RESISTCOLD,TRAIT_RESISTHIGHPRESSURE,TRAIT_RESISTLOWPRESSURE,TRAIT_NOFIRE,TRAIT_CHUNKYFINGERS, TRAIT_CLUMSY, TRAIT_CLOWNLIKE, TRAIT_RADIMMUNE,TRAIT_GENELESS,TRAIT_PIERCEIMMUNE,TRAIT_NODISMEMBER)
```

Comment:
as compensation for this nerf, could you also make it so that people slip when they walk over prone bananium golems, just like how they slip when they walk over a prone person who's wearing a clown PDA and clown shoes?

Reply:
I mean while this is a perfectly fine and legit suggestion I'll look into, I have to genuinely question why you would ever, ever take being a bananium golem and expect to be capable for fighting opponents as successfully as any other golem. 

I don't think I actually give a rats arse how good they are at fighting, that's ultimately missing the point, so neither should you. It's actually criminal they ever were. But making them slippery is consistent with them being made of bananium, so I'm on board with it.